portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```

Comment:
Method names should be descriptive of the purpose - since you've changed what this method does, should also change the method name. In fact, it bothers me that this was called **add**RandomGreeting in the first place, because it's not "adding" anything, but "getting" :) (but I know that you didn't write that). Anyhow, suggest "getRandomFact" or something along those lines.

Reply:
I agree about how they probably should never have named this method using the word "add" in the first place since it is really replacing the text.

I'd also say though that I feel like "get" usually implies that a function is returning a value, and the button in this case isn't returning anything but rather modifying the content of the UI.  So I'd personally go with something like updateRandomGreeting, or replaceRandomGreeting, or changeRandomGreeting.. or really anything that you want to make an argument for =)

Reply:
got it!